Encodings.md

```diff
@@ -247,6 +253,15 @@ and handled as wrapping around in 2's complement notation so that the original
 values are correctly restituted. This may require explicit care in some programming
 languages (for example by doing all arithmetic in the unsigned domain).
 
+One strategy that might be employed to avoid the above mentioned overflow is to
```

Comment:
This is a rather wordy addition. I think the spec should remain concise. If we want to elaborate on this, we should move the discussion of signedness and bit width to a dedicated "Pitfalls" subsection, IMHO.

Reply:
Sorry, I do that 😅. This can be reduced to a single sentence if we decide to mandate the use of no more bits than the physical type.